shell/app/org-home/stores/org.tsx

```diff
@@ -115,6 +115,7 @@ const org = createStore({
               const menusMap = getSubSiderInfoMap();
               const appCenterAppList = getAppCenterAppList();
               appCenterAppList.forEach((a) => { appMap[a.key] = a; });
+              layoutStore.reducers.clearLayout();
```

Comment:
have to clear too in other modules such as admin?

Reply:
Switching organization didn't affect admin module, and fdp module will register itself.